File: FungeeCommandsManager/src/main/java/org/openredstone/commands/FunCommand.java
package org.openredstone.commands;

import net.md_5.bungee.api.plugin.Command;
import org.openredstone.FungeeCommandsManager;

abstract class FunCommand extends Command {
    FunCommand(String name) {
        super(name, FungeeCommandsManager.permissionFor(name));
    }
}

File: fungee-manager/src/main/kotlin/org/openredstone/fungee/manager/commands/FunCommand.kt
package org.openredstone.fungee.manager.commands

import net.md_5.bungee.api.plugin.Command
import org.openredstone.fungee.manager.FungeeCommandsManager

abstract class FunCommand(plugin: FungeeCommandsManager, name: String) :
    Command(name, plugin.permissionFor(name))
